src/equations/shallow_water_2d.jl

```diff
@@ -323,6 +390,33 @@ Details are available in Eq. (4.1) in the paper:
   Well-balanced and energy stable schemes for the shallow water equations with discontinuous topography
   [DOI: 10.1016/j.jcp.2011.03.042](https://doi.org/10.1016/j.jcp.2011.03.042)
 """
+@inline function flux_fjordholm_etal(u_ll, u_rr, orientation::Integer, equations::ShallowWaterEquations2D)
```

Comment:
The argument in the docstring above  should become `orientation_or_normal_direction`

Reply:
Done